earth_enterprise/src/fusion/autoingest/plugins.src/RasterProject.src

```diff
@@ -1157,9 +1165,13 @@ uint32 RasterProjectAssetVersionImplD::BuildIndex(
       genInfo[i].PopulatePacketLevels(packgen);
     }
 
-#pragma omp critical 
-    {
-      // remember this new PacketGen version
+    // remember this new PacketGen version
+    prelim_packgenvers[i] = packgen;
+  }
+
+  // Populate packgenkids, packgenassets, and packgenvers from prelim_packgenvers
+  for(auto & packgen : prelim_packgenvers) {
```

Comment:
I'm just thinking out loud; there's no need to take any action, but there's got to be some algorithm for combining vectors in parallel.